src/format.rs

```diff
@@ -85,11 +85,11 @@ impl<'a> Formatter<'a> {
         for statement in module.statements.iter() {
             let start = statement.location().start;
             match statement {
-                Statement::Import { .. } => {
+                Statement::Import { module, .. } => {
                     has_imports = true;
                     let comments = self.pop_comments(start);
                     let statement = self.statement(statement);
-                    imports.push(commented(statement, comments))
+                    imports.push((module.join(""), commented(statement, comments)))
```

Comment:
Slices implement Ord (https://doc.rust-lang.org/std/slice/index.html#trait-implementations-1) so I think we can avoid joining the vec here, removing those allocations.

Reply:
Did, not know that. Sweet!